api/src/main/java/ca/bc/gov/educ/api/ruleengine/util/Transformer.java

```diff
@@ -0,0 +1,23 @@
+package ca.bc.gov.educ.api.ruleengine.util;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import java.io.InputStream;
+
+public interface Transformer {
```

Comment:
No need for this interface if we are removing logClientHttpReqResponseDetails in LogHelper Class